lib/dndheader.sty

```diff
@@ -3,26 +3,56 @@
 % Setup for custom footer
 \pagestyle{fancy}
 
-\fancyhfoffset[LE]{28pt} % push the footer left on even pages so it fits the decal
-\fancyhfoffset[RO]{30pt} % push the footer right on odd pages so it fits the decalbg-img
+\newlength{\nobgfooterheight}
+\setlength{\nobgfooterheight}{\paperheight}
+\addtolength{\nobgfooterheight}{-1in}
+\addtolength{\nobgfooterheight}{-\topmargin}
+\addtolength{\nobgfooterheight}{-\headheight}
+\addtolength{\nobgfooterheight}{-\headsep}
+\addtolength{\nobgfooterheight}{-\textheight}
+\addtolength{\nobgfooterheight}{-\footskip}
+
 \renewcommand{\headrulewidth}{0.0pt} %no rule for header
 \renewcommand{\footrulewidth}{0.0pt} %no rule for footer
 
-\fancyhead{} % clear all header fields
-\fancyfoot{} % clear all footer fields
+\fancyhf{} % clear all headers and footers
+
+\fancyhead{
```

Comment:
Attaching the background to the header is clever. Eliminates the `background` package dependency.